### ISSUE TITLE
Input, Input Message - Ensure all interactive functionality is operable with the keyboard - (2036569943)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5576]
### Module: 
 16 Input &amp; Input Message 

### Violation: 
 Ensure all interactive functionality is operable with the keyboard 

![image](https://user-images.githubusercontent.com/5023024/202763878-568ee1ea-e366-483f-89a5-9cb6eb822c36.png)

### WCAG Reference: 
- [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) 

### Instance ID: 
 2036569943 

### Severity: 
 8 

### Description: 
 [Issue]
There are buttons present besides the below mentioned input fields that cannot be navigated to and/or operated with the keyboard alone. Examples include:

- Vertical:
- Vertical with Prefix and Suffix text:
- Horizontal:
- Horizontal with Prefix and Suffix text:
- Number input error message

[User Impact]
Keyboard-only users will be prevented from accessing the functionality provided by these controls.

[Code Reference] 
 ```
<button class="number-button-item" data-adjustment="up" tabindex="-1" type="button"><calcite-icon aria-hidden="true" icon="chevron-up" scale="s" calcite-hydrated=""></calcite-icon></button>
 ``` 

### Note: 
 [Suggestion]
Ensure all interactive functionality is operable with the keyboard. The best way to accomplish this is by using appropriate native controls, which come with keyboard functionality built in.

Remove tabindex="-1" from the button elements.

[Compliant Code Example] 
 ```
<button class="number-button-item" data-adjustment="up" type="button"><calcite-icon aria-hidden="true" icon="chevron-up" scale="s" calcite-hydrated=""></calcite-icon></button>
 ```
 

### Media Type: 
 Keyboard Accessibility 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
The up and down arrows don't function very well with JAWS active. 

If assistive technology users can use the arrow keys **-OR-** access the buttons to change the values we'd have a resolution.

**Comment by geospatialem** [comment]
Not applicable, as this is expected behavior in the native input.
